examples/with-antd-mobile/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "with-antd-mobile",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "dependencies": {
-    "antd-mobile": "^1.1.2",
-    "babel-plugin-import": "^1.1.1",
+    "antd-mobile": "1.4.0",
```

Comment:
@cncolder 
Next time please remember do not  lock antd-mobile version, just use `antd-mobile@1.x`.